pkg/platform/platform.go

```diff
@@ -58,6 +58,10 @@ func CommonControlPlaneCharts() []helm.LokomotiveChart {
 			Name:      "bootstrap-secrets",
 			Namespace: "kube-system",
 		},
+		{
```

Comment:
`packet-ccm` is not a _Common_ control plane chart. It is Packet platform specific. This won't work on other platforms.